app/src/components/Modals/add-user-modal.tsx

```diff
@@ -140,62 +148,79 @@ const AddUserModal: FC<AddUserModalProps> = ({ isOpen, onClose, userInfo }) => {
             justifyContent="center"
             fontWeight="semibold"
             fontSize="headline.sm"
+            fontFamily="heading"
             lineHeight="32"
             padding="24px"
-            borderBottom={"1px solid #fafafa"}
+            borderBottomWidth="1px"
+            borderStyle="solid"
+            borderColor="border.neutral"
           >
-            Add User
+            {t("add-user")}
           </ModalHeader>
           <ModalCloseButton marginTop="10px" />
-          <ModalBody>
+          <ModalBody p="24px" px="48px">
             <form onSubmit={handleSubmit(onSubmit)}>
               <Box display="flex" flexDirection="column" gap="24px">
                 <FormInput
                   id="name"
                   error={errors.name}
-                  label="Full Name"
+                  label={t("full-name")}
                   register={register}
                 />
                 <FormInput
                   id="email"
                   error={errors.email}
-                  label="Email"
+                  label={t("email")}
                   register={register}
                 />
 
                 <FormSelectInput
-                  label="Role"
+                  label={t("role")}
                   value={inputValue}
                   register={register}
                   error={errors.role}
                   id="role"
                   onInputChange={onInputChange}
                 />
                 <FormSelectOrganization
-                  label="Is organization"
+                  label={t("is-organization")}
                   value={inputValue}
                   register={register}
                   error={errors.role}
                   id="isOrganization"
                   onInputChange={onInputChange}
                 />
-                <Button
-                  h="56px"
-                  paddingTop="16px"
-                  paddingBottom="16px"
-                  px="24px"
-                  letterSpacing="widest"
-                  textTransform="uppercase"
-                  fontWeight="semibold"
-                  fontSize="button.md"
-                  w="100%"
-                  type="submit"
-                >
-                  Add user
-                </Button>
               </Box>
             </form>
           </ModalBody>
+          <ModalFooter
+            borderTopWidth="1px"
+            borderStyle="solid"
+            borderColor="border.neutral"
+            w="full"
+            display="flex"
+            alignItems="center"
+            p="24px"
+            justifyContent="center"
+          >
+            <Button
+              h="56px"
+              w="472px"
```

Comment:
Do these need to be hardcoded? That usually conflicts with people setting custom font sizes in their browser/ OS, which then breaks the layout instead of just scaling the button.